test/cljam/io/cram/encode/record_test.clj

```diff
@@ -213,7 +233,7 @@
 
       (is (= 1 (count (get ds-res :TL))))
       (is (= 13 (get-in ds-res [:TL 0 :content-id])))
-      (is (= [1 1 1 1 0] (seq (get-in ds-res [:TL 0 :data]))))
+      (is (= [0 0 0 0 1] (seq (get-in ds-res [:TL 0 :data]))))
```

Comment:
Why expected data-series-encodings in :TL tag is reversed? I can't find it reasons.

Reply:
The tag dictionary builder assigns sequential IDs in the order it encounters each unique combination of tags for the first time. In this test case, the combination of tags `RG:Z,MD:Z,NM:c` is assigned ID `0` first, followed by the empty tag combination, which is assigned ID `1`.

On the other hand, before this PR the tag dictionary was crafted by hand (with the intention of following lexicographical order), so the order of entries was different than how it is now. I think this difference is trivial.

Reply:
Thank you for the comment. I understand the difference.